src/components/Burst.css

```diff
@@ -60,3 +60,35 @@
   background: #484848;
   border-color: #e79600;
 }
+
+.burstInfoButton {
+  height: 20px;
+  margin: 20px;
+}
+
+.busrtInfoButton:hover {
+  cursor: pointer;
+}
+
+.burstInformation {
+  background-color: #fff;
+  color: black;
+  font-size: 14px;
+  padding: 10px;
+  border-radius: 5px;
+  overflow: scroll;
+  margin: 0px 20px;
+  height: 80px;
```

Comment:
the height of the box could be bigger for readability. 
we could make the height of the box a bit more dynamic using calc() in css
```
max-height: calc(100vh - 600px);
```